src/main/java/org/takes/rq/RqFake.java

```diff
@@ -73,7 +73,8 @@ public RqFake(final String method, final String query) {
      * @param query HTTP query
      * @param body HTTP body
      */
-    public RqFake(final String method, final String query, final String body) {
+    public RqFake(final CharSequence method, final CharSequence query,
+                  final CharSequence body) {
```

Comment:
@yegor256 during last review, @pinaf asked me to fix checkstyle violations. 
[ERROR] /src/main/java/org/takes/rq/RqFake.java[76]: Line is longer than 80 characters (found 85). (LineLengthCheck)
I change style of method declaration - add "new line char" after "query,". I like when parameters of method starts on one vertical line, as for me it is improve readiness of code. 

As I understand you want me to do like like here  /org/takes/facets/fork/FkParams.java:132
I will fix it